modules/core/update/src/FarmUpdate.php

```diff
@@ -76,32 +76,25 @@ public function rebuild(): void {
   }
 
   /**
-   * Lists excluded config items.
+   * Lists managed config items.
    *
-   * Lists config items that should be excluded from all automatic updates.
+   * Lists config items that should be automatically updated.
    *
    * @return array
    *   An array of config item names.
    */
-  protected function getExcludedItems() {
+  protected function getManagedConfig() {
 
-    // Ask modules for config exclusions.
-    $exclude_config = $this->moduleHandler->invokeAll('farm_update_exclude_config');
+    // Ask modules for managed configuration items.
+    $managed_config = $this->moduleHandler->invokeAll('farm_update_managed_config');
```

Comment:
Just trying to think if there is still a use-case for `farm_update_exclude_config`... Currently you can keep the `farm_update` module on and exclude a select few config items (including from other modules) that you want to manage yourself. With this change to opt-in that wouldn't be possible.

But instead of keeping `farm_update_exclude_config` a much simpler solution might be to simply add an alter hook `farm_update_managed_config_alter` so you could _remove_ some config from otherwise being included.

Reply:
I almost did this when I was drafting this PR - it's an obvious next step - and very simple. :smile: 

I'll add a commit with tests for altering "managed" config...